CHANGELOG.md

```diff
@@ -143,13 +143,13 @@
 
 ### Minor Changes
 
-- [Added] - Render an input inside of pie-input with a type property that defaults to text ([#1138](https://github.com/justeattakeaway/pie/pull/1138)) by [@jamieomaguire](https://github.com/jamieomaguire)
+- [Added] - Render an input inside of pie-text-input with a type property that defaults to text ([#1138](https://github.com/justeattakeaway/pie/pull/1138)) by [@jamieomaguire](https://github.com/jamieomaguire)
 
 ## 2.48.0
 
 ### Minor Changes
 
-- [Added] - New pie-input component with boilerplate code ([#1135](https://github.com/justeattakeaway/pie/pull/1135)) by [@jamieomaguire](https://github.com/jamieomaguire)
+- [Added] - New pie-text-input component with boilerplate code ([#1135](https://github.com/justeattakeaway/pie/pull/1135)) by [@jamieomaguire](https://github.com/jamieomaguire)
```

Comment:
I'm on the fence about revising the changelogs when they were accurate the time

Reply:
agreed - I asked in the team channel and the only response I got was to update them